specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-02-01-preview/sensors.json

```diff
@@ -0,0 +1,507 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Downloaded reset password file",
+            "schema": {
+              "type": "object",
+              "format": "file"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.IoTSecurity/sensors/{sensorName}/triggerTiPackageUpdate": {
+      "post": {
+        "x-ms-examples": {
+          "Trigger threat intelligence package update": {
+            "$ref": "./examples/Sensors/TriggerTiPackageUpdate.json"
+          }
+        },
+        "tags": [
+          "Sensors"
+        ],
+        "operationId": "Sensors_TriggerTiPackageUpdate",
+        "description": "Trigger threat intelligence package update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SensorScope"
+          },
+          {
+            "$ref": "#/parameters/SensorName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Triggered threat intelligence package update"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SensorsList": {
+      "description": "List of IoT sensors",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List data",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SensorsModel"
+          }
+        }
+      }
+    },
+    "SensorsModel": {
+      "type": "object",
+      "description": "IoT sensor model",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "IoT sensor properties",
+          "$ref": "#/definitions/SensorProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "SensorProperties": {
+      "type": "object",
+      "description": "IoT sensor properties",
+      "properties": {
+        "connectivityTime": {
+          "description": "Last connectivity time of the IoT sensor",
+          "readOnly": true,
+          "type": "string",
+          "example": "2020-11-17T12:31:25Z"
+        },
+        "creationTime": {
```

Comment:
Why is this provided separately from SystemData?

Reply:
The creationTime field was added before the SystemData became required. Removing it would be a breaking change.